### ISSUE TITLE
Image error 

### ISSUE THREAD

**Issue by adelrockz** [thmsgbrt/thmsgbrt#16]
![image](https://github.com/user-attachments/assets/b7916305-5f45-4ec8-85ee-7682cd45764e)

**Comment by Rishiofficial432-432** [comment]
yeah so the instagram images aren't showing up because the puppeteer scraper is broken. instagram changes their site structure all the time and has pretty strict bot detection, so this kind of thing breaks often.

looks like puppeteerService.getLatestInstagramPostsFromAccount is returning empty values which is why you're getting those blank image tags.

couple ways to fix:

1. switch to instagram's actual API - they have a Basic Display API that's free for basic stuff. bit of setup but way more stable than scraping

2. fix the puppeteer selectors - instagram probably changed their html since this was written. you'd need to update puppeteer.service.js with the current selectors

3. use a third party service that handles instagram feeds - stuff like juicer or curator. they deal with instagram's changes so you don't have to

4. or just remove it/use placeholder images for now if it's not critical

honestly the API route is probably best if you want something that won't break every few months. happy to help if you need it
